NuRadioReco/framework/parameters.py

```diff
@@ -39,7 +39,13 @@ class channelParameters(Enum):
     cr_xcorrelations = 8  # dict of result of crosscorrelations with cr templates
     nu_xcorrelations = 9  # dict of result of crosscorrelations with nu templates
     signal_time = 10  # the time of the maximum amplitude of the envelope
-    noise_rms = 11  # the root mean square of the noise
+    signal_regions = 11     # list of start and end times of regions that likely contain a signal
+    noise_regions = 12      # list of start and end times of regions that likel do not contain any signals
+    noise_rms = 13  # the root mean square of the noise
```

Comment:
isn't it dangerous to rename the index of the enums? Would't this screw up existing data files as existing noise_rms values would be interpreted as `signal_regions`? I think the rule is: Never change an index, only add or delete  indices.

Reply:
Yes, I shouldn't have done that.